assets/dev/js/editor/controls/color.js

```diff
@@ -1,5 +1,6 @@
 import ControlBaseDataView from './base-data';
 import ColorPicker from '../utils/color-picker';
+import GlobalControlSelect from '../../../../../core/kits/assets/js/globals/global-select-behavior';
```

Comment:
It can't be used in the kits manager.js
The manager runs before controls even exist. there is no way to attach the behavior only to the color or typography controls

Reply:
Done